### ISSUE TITLE
SteamVR home flickers a lot

### ISSUE THREAD

**Issue by DataBeaver** [ValveSoftware/SteamVR-for-Linux#82]
#### Your system information
* Steam client version: 2017-12-01 15:05:56
* Distribution: Debian
* Graphics driver version: 387.22
* Gist for SteamVR System Information: https://gist.github.com/DataBeaver/4e11da6a3dd0d765267c1a4eb4b43842
* Opted into Steam client beta?: yes
* Opted into SteamVR beta?: yes
* Have you checked for system updates?: yes

#### Please describe your issue in as much detail as possible:
When running SteamVR home, the screen flickers at a rate of 1-2 Hz and briefly displays an image at a significant offset.  I cannot easily tell if it's translation, rotation or both.  It's somewhat content dependent; in the summit pavilion it occurs in most places but if I move to the isolated spot in the garden and look to the right of the house, the flickering disappears.  On Mars I rarely see any flickering.

Frame timings show distinct spikes while the flickering is occurring:
![steamvr_frame_timing](https://user-images.githubusercontent.com/1126732/33530276-ec868f24-d885-11e7-90d6-8387d41eb6f4.png)

**Comment by ChristophHaag** [comment]
Related: https://github.com/ValveSoftware/SteamVR-for-Linux/issues/68

Are these spikes to related to very regularly moving the headset back and forth into a specific direction or does it look like this when it's not moving too?

**Comment by DataBeaver** [comment]
It looks like this when I'm holding the headset still.  There's bound to be some noise from my muscles, especially since I held the headset in only one hand while taking the screenshot, but I'm not making any regular movements on purpose.

**Comment by Plagman** [comment]
Just to double-check, does it get better if you set PowerMizer to "prefer maximum performance" in nvidia-settings?

**Comment by DataBeaver** [comment]
Nope, PowerMizer setting has no appreciable effect.  I suppose I could try counting flickers per minute to obtain an objective result, but subjectively it isn't any better.

**Comment by Olm-e** [comment]
Hi, on my side, with drivers **387.34**, I get really unstable and flickering (unusable)result if powermizer is on Auto mode, 
and **correct rendering speed with it set to maximum performance**, 
linux mint 18.2 with laptop i7 and nvidia 1060
it is same for SteamVR or/and OpenVR app like Godot engine with openvr gdnative
tks for pointing this !

**Comment by danginsburg** [comment]
I have this repro'd here.  I don't have it root caused yet, but it appears to be due to an interaction between our adaptive sampling method and incorrect GPU timestamps being reported on the vrcompositor.  As a temporary workaround, you could launch SteamVR Home without auto fidelity enabled:

~/.local/share/Steam/steamapps/common/SteamVR/tools/steamvr_environments/game$ ~/.steam/steam/ubuntu12_32/steam-runtime/run.sh ./steamtours.sh -vulkan -vr -noautofidelity

The problem also happens on 387.34 (not just 387.22) and I don't believe it is related to PowerMizer settings (at least on my machine).
